cmd/bblfshd/main.go

```diff
@@ -0,0 +1,158 @@
+package main
+
+import (
+	"flag"
+	"net"
+	"os"
+	"os/signal"
+	"strings"
+	"syscall"
+
+	"github.com/bblfsh/server/daemon"
+	"github.com/bblfsh/server/runtime"
+
+	"github.com/sirupsen/logrus"
+	"google.golang.org/grpc"
+	"gopkg.in/bblfsh/sdk.v1/sdk/server"
+)
+
+var (
+	version = "undefined"
+	build   = "undefined"
+
+	network        *string
+	address        *string
+	storage        *string
+	transport      *string
+	maxMessageSize *int
+	log            struct {
+		level  *string
+		format *string
+		fields *string
+	}
+
+	logger server.Logger
+)
+
+func init() {
+	cmd := flag.NewFlagSet("server", flag.ExitOnError)
+	network = cmd.String("network", "tcp", "network type: tcp, tcp4, tcp6, unix or unixpacket.")
+	address = cmd.String("address", "localhost:9432", "address to listen.")
+	storage = cmd.String("storage", "/var/lib/bblfshd", "path where all the runtime information is stored.")
```

Comment:
It would be nice to have an option to not cache the images; it would be very useful when you're debugging interactions between drivers and the servers and you change the local image frequently. But that's outside the scope of this PR and could be done later.

Reply:
The images should be stored and updated if need it. A feature to be develop is a command line to to force this update in the server, as `docker pull <image>`